utils/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/source/KafkaSourceFactory.scala

```diff
@@ -159,6 +161,12 @@ object KafkaSourceFactory {
 
   case class KafkaSourceFactoryState[K, V](contextInitializer: ContextInitializer[ConsumerRecord[K, V]])
 
+  case class KafkaTestParametersInfo(uiParameters: List[Parameter], createTestRecord: Any => TestRecord)
```

Comment:
[//]: # (Let's call it similar to `TestWithParametersSupport.testParametersDefinition`. Maybe `parametersDefinition`?

Reply:
Renamed :+1: